webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -54,14 +62,20 @@ const DEFAULT_MODAL_STATE = {
   modelName: '',
   tags: [],
   visible: false,
+  workspace: '1',
 };
 
 const useModalModelCreate = ({ onClose }: Props = {}): ModalHooks => {
+  const { canViewModelWorkspace } = usePermissions();
+  const loadableWorkspaces = useWorkspaces();
+  const workspaces = Loadable.match(loadableWorkspaces, {
```

Comment:
Noting that `workspaces` needs to be filtered for only the workspaces that a user has access to.